src/Models/RouteStatistic.php

```diff
@@ -85,6 +85,7 @@ public function getLogAttributes(Request $request, $response, ?int $time = null,
             'team_id' => $this->getRequestTeam($request)?->getKey(),
             'method' => $request->getMethod(),
             'route' => $request->route()?->getName() ?? $request->route()?->uri(),
+            'parameters' => config('route-statistics.store_parameters') ? json_encode($request->route()->originalParameters()) : null,
```

Comment:
Is there a reason for not using the[ Array casting](https://laravel.com/docs/11.x/eloquent-mutators#array-and-json-casting) instead of manual json_encoding here?

Reply:
You're right, it is already supported in Laravel 8 so I can indeed use array casting

Reply:
As it turns out console table doesn't like an array, so I need to map each table element. And I now remember WHY the select is in place

```php
$results = $query
    ->limit($this->option('limit'))
    ->get()
    ->map(function(RouteStatistic $item) {
        $data = $item->toArray();
        if (config('route-statistics.store_parameters')) {
            $data['parameters'] = json_encode($data['parameters']);
        } else {
            unset($data['parameters']); // Hide parameters if not supported in config
        }
        return $data;
    });
```

Reply:
@svenbw I still do not get why we are json encoding here? I mean this function should just return an array of parameters - it is okay if those are nested arrays, the model should take care of storing this correctly.

Reply:
After the requested to not use `json_encode` to store the data in the model I casted the data using the cast functionality of the model. When the array with data is requested the parameters attribute will return an array.
That part of code is used to cast the data from an array to something that can be printed in the console.